### DISCUSSION TITLE
Remote Database Providers

### DISCUSSION THREAD

**Discussion by HellsAvalanche** [vrcx-team/VRCX#439] [General]
Lemme start this out with I already have two! Didn't spot something like this on the discussion board, so I figured I'd spark one up!
I have access to just-h and ares! Anyone got any others they wanna share around?

https://avtr.just-h.party/vrcx_search.php
https://api.ares-mod.com/vrcx_search.php

**Comment by Myrkie** [comment]
the only other one that exists that im aware of is my own, which has been absorbed by just-h.party already.
